components/icon/badge-icon/check-props.js

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
```

Comment:
This is a new component.

Reply:
I have deleted the checkProps file, it was not imported anywhere.